store/store.go

```diff
@@ -1207,13 +1211,21 @@ func (s *Store) checkCandidates(currentSnaps []*currentSnapJSON, user *auth.User
 	return updateData.Payload.Packages, nil
 }
 
+func isActuallyUpdate(remote *snapDetails, installed *RefreshCandidate) bool {
+	rrev := snap.R(remote.Revision)
+	return !(rrev == installed.Revision || findRev(rrev, installed.Block))
+}
+
+var refreshForCandidates = (*Store).refreshForCandidates
+
+// LookupRefresh returns an updated snap given its refresh candidate
```

Comment:
"the store offered snap revision given its refresh candidate"?   also missing a full-stop